README.md

```diff
@@ -1,99 +1,122 @@
+~3.1.~ `$ brew install jenv`
 
-To run the service, ensure there is an instance of Postgres running and then run
+Depending if you use Bash or Zsh, run the right command for you :
+> Bash
+`$ echo 'export PATH="$HOME/.jenv/bin:$PATH"' >> ~/.bash_profile`
+`$ echo 'eval "$(jenv init -)"' >> ~/.bash_profile`
 
-```SPRING_PROFILES_ACTIVE=local ./gradlew bootRun```
+> Zsh (The default interactive shell for docker)
+`$ echo 'export PATH="$HOME/.jenv/bin:$PATH"' >> ~/.zshrc`
+`$ echo 'eval "$(jenv init -)"' >> ~/.zshrc`
 
-Dependencies
----
-The service has an attached Postgres database as well as several back ends the details of which can be found in the [docker-compose.yml](docker-compose.yml) file.
+#### 4. Close your terminal to finalise jEnv set up.
 
-To run against local Dockerised back-ends and database
+#### 5. Reopen a new window and run :   
+~5.1.~ `$ /usr/libexec/java_home -V` 
+~5.2.~ `$ jenv add <choose one/all the versions you have found when you ran the previous command>`
+*E.g.: `jenv add /Library/Java/JavaVirtualMachines/openjdk-11.0.2.jdk/Contents/Home`*
 
-```docker-compose up```
 
-Once the database container is running, initialise the application database schemas
+## Dependencies
 
-```initSchema.sh```
+The service has an attached Postgres database as well as several back-ends. The details of which can be found in the `docker-compose.yml` file.
+:::info
+⚠️ To open all of these, complete the Building and Running section.
+:::
 
-There are also Wiremock stubs for each of the back end calls which the `test` Spring profile runs against, to run these use the following command along with `docker-compose up`
+**Default port**
+Starts the application on port '8080'. To override, set server.port (eg SERVER_PORT=8099 java -jar etc etc)
 
-```bash runMocks.sh```
+**Swagger UI**
+The project builds swagger specifications which can be examined and tested via the Swagger UI. http://localhost:8080/swagger-ui.html
 
-### Default port
-Starts the application on port '8080'.
-To override, set server.port (eg SERVER_PORT=8099 java -jar etc etc)
+**Application health** : `$ curl -X GET http://localhost:8080/health`
+**Application info** : `$ curl -X GET http://localhost:8080/info`
+**Application Ping** : `$ curl -X GET http://localhost:8080/ping`
+**Application Feature Flags** : `$ curl -X GET http://localhost:8080/feature-flags`
+**To Check dependency versions** : `$ ./gradlew dependencyUpdates`
 
-### Swagger UI
-The project builds swagger specifications which can be examined and tested via the Swagger UI.
+#### *Flyway commands*
+**Migrate database** : `$ gradle flywayMigrate -i`
+**View details and status information about all migrations** : `$ gradle flywayInfo`
+**Baseline an existing database, excluding all migrations up to and including baselineVersion** :  `$ gradle flywayBaseline`
```

Comment:
Because Spring Boot automatically bootstraps our database we don't generally use the Gradle commands much (though it's still useful to have) - the one exception to this is `flywayClean` which cleans away all migrations applied to a database and removes the flyway schema. This can be useful if you somehow manage to corrupt your migrations, which I've managed a fair few times 😄 . 

Worth noting that we usually use the provided `./gradlew` (Gradle wrapper) file rather than `gradle`. This ensures you've always got the right version of gradle for a given repo but is otherwise identical in usage.

```suggestion
**Baseline an existing database, excluding all migrations up to and including baselineVersion** :  `$ ./gradlew flywayBaseline`
**Clean schema** : `$ ./gradlew flywayClean`
```